ros_unreal_interface/ConnectionMessage message		# data about connection settings
---
bool ack 			# acknowledged informations